templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -1026,14 +1039,57 @@ Resources:
         Statement:
           - Effect: "Allow"
             Principal:
-              AWS: !GetAtt BitbucketClusterNodeRole.Arn
-            Action: "es:*"
+              AWS: "*"
```

Comment:
Isn't this too wide? What s the effective implication of this policy?

Reply:
This is actually less wide than it was before but enforces that the app can only interact with the elasticsearch cluster via the HTTP API (i.e. it doesn't have permission to create/delete elasticsearch clusters). Although, I wonder if this breaks DIY backup 🤔

Reply:
Does the Elasticsearch restore work in the current implementation? I remember some tickets around it.

Reply:
I don't think so. I'm pretty sure steve looked into it and it did nothing.

Reply:
The DIY backup scripts only use the HTTP API so the permissions will be fine. However, they use request signing for AWS elasticsearch backup/restore instead of basic auth so it will probably break the DIY backup/restore 🤔

Reply:
Actually I think it should be fine, they should be able to use the S3 configuration. Also, the DIY backup scripts are  not really the best backup option on AWS.